nixos/modules/services/networking/syncplay.nix

```diff
@@ -7,6 +7,7 @@ let
 
   cmdArgs =
     [ "--port" cfg.port ]
+    ++ cfg.extraArgs
     ++ optionals (cfg.salt != null) [ "--salt" cfg.salt ]
     ++ optionals (cfg.certDir != null) [ "--tls" cfg.certDir ];
```

Comment:
```suggestion
  cmdArgs = [
    "--port" cfg.port
  ] ++ optionals (cfg.salt != null) [ "--salt" cfg.salt ]
    ++ optionals (cfg.certDir != null) [ "--tls" cfg.certDir ]
    ++ cfg.extraArgs;
```

Reply:
Using *nixpkgs-fmt* would move lines 11 and 12 one indent further out:

```patch
diff --git a/nixos/modules/services/networking/syncplay.nix b/nixos/modules/services/networking/syncplay.nix
index bf55a904d37..4b70cea543d 100644
--- a/nixos/modules/services/networking/syncplay.nix
+++ b/nixos/modules/services/networking/syncplay.nix
@@ -5,11 +5,12 @@ with lib;
 let
   cfg = config.services.syncplay;

-  cmdArgs =
-    [ "--port" cfg.port ]
-    ++ cfg.extraArgs
-    ++ optionals (cfg.salt != null) [ "--salt" cfg.salt ]
-    ++ optionals (cfg.certDir != null) [ "--tls" cfg.certDir ];
+  cmdArgs = [
+    "--port"
+    cfg.port
+  ] ++ optionals (cfg.salt != null) [ "--salt" cfg.salt ]
+  ++ optionals (cfg.certDir != null) [ "--tls" cfg.certDir ]
+  ++ cfg.extraArgs;

 in
 {
```

Whereas the only moving the *extraArgs* line to the end would still keep the port-related list in a single line if written like this, so I'd go for this one for legibility (and also consistency since the lists used for `optionals` are on one line too):

```patch
diff --git a/nixos/modules/services/networking/syncplay.nix b/nixos/modules/services/networking/syncplay.nix
index dc2401533e2..73e1460fa67 100644
--- a/nixos/modules/services/networking/syncplay.nix
+++ b/nixos/modules/services/networking/syncplay.nix
@@ -7,9 +7,9 @@ let

   cmdArgs =
     [ "--port" cfg.port ]
-    ++ cfg.extraArgs
     ++ optionals (cfg.salt != null) [ "--salt" cfg.salt ]
-    ++ optionals (cfg.certDir != null) [ "--tls" cfg.certDir ];
+    ++ optionals (cfg.certDir != null) [ "--tls" cfg.certDir ]
+    ++ cfg.extraArgs;

 in
 {
```

(I also threw nixpkgs-fmt against the file once, adding minor changes in unrelated places)

Reply:
fine for me